test/test_jit_py3.py

```diff
@@ -533,6 +534,28 @@ def forward(self, x):
         # Check that ignored method is still intact.
         self.assertEqual(inp, n.ignored_method(inp))
 
+    def test_any_return_type_disallowed(self):
+        class Mod(nn.Module):
+            def __init__(self):
+                super().__init__()
+
+            def forward(self, inp: torch.Tensor) -> Any:
+                return inp * inp
+
+        def any_function(inp: torch.Tensor) -> Any:
+            return inp * inp
+
+
+        with self.assertRaisesRegex(Exception, "Return type annotation of Any is not permitted"):
```

Comment:
I tried to use `assertRaisesRegexWithHighlight` for this but it couldn't find my highlight even though I tried this out on the test case from the issue and saw that the right section was highlighted.

cc: @gmagogsfm

Reply:
thanks for letting me know, so just running this testcase should repro?

Reply:
Yes, but change this test to 
```suggestion
        with self.assertRaisesRegexWithHighlight(Exception, "Return type annotation of Any is not permitted", "Any"):
```